front/src/components/QrReader/QrReaderContainer.tsx

```diff
@@ -73,11 +74,24 @@ function QrReaderContainer({ onSuccess }: IQrReaderContainerProps) {
   const onScan = async (qrReaderResultText: string, multiScan: boolean) => {
     if (!isProcessingQrCode) {
       setIsProcessingQrCode(true);
+      setBoxNotOnwned("");
       const qrResolvedValue: IQrResolvedValue = await resolveQrCode(
         qrReaderResultText,
         multiScan ? "cache-first" : "network-only",
       );
       switch (qrResolvedValue.kind) {
+        case IQrResolverResultKind.BOX_NO_PERMISSION: {
```

Comment:
pls remove this case and rather call a trigger_error in the QRResolver hook that shows than a toast.